pkg/datastore/example_multiinstance_test.go

```diff
@@ -99,8 +99,7 @@ func ExampleDataStore_multiInstance() {
 	// [Dev/P1337] Bob: 31 <nil>
 	// [Prod/P1337] John: 36 <nil>
 	// [Dev/P3] Pat: 39 <nil>
-	// Unable to locate record [record=[/P3] : 0]
-	//	record not found
+	// err != nil - true
```

Comment:
After I started adding DB name to errors, the message being printed out became "Unable to locate record [record=[/P3] : 0, **dbName=\<database name\>**]". Database name is going to be different depending on the value of DB_NAME env. var. So, the message printed out is not guaranteed to be the same in all environments. In order for the testable example to pass, this comment needs to be exactly the same as what's gonna be actually printed (so, the log messages being printed have to be fixed). As a result, I needed to make a change on line #81

Reply:
@jeyhun you can try printing partially or use the dbname, not sure what the issue as the Examples are full fledged tests without any external callers